server/player/src/packet_handlers/block_interaction/crafting_table.rs

```diff
@@ -0,0 +1,33 @@
+use super::InteractionHandler;
+
+use feather_core::network::packets::OpenWindow;
+use feather_core::text::TextRoot;
+use feather_server_types::{Game, Network};
+use fecs::{Entity, World};
+
+pub struct CraftingTableInteraction;
+
+impl InteractionHandler for CraftingTableInteraction {
+    fn handle_interaction(
+        &self,
+        _game: &mut Game,
+        world: &mut World,
+        player: Entity,
+        window_id: u8,
+    ) {
+        // Show the crafting table GUI for the player
+        let open_window_packet = Box::new(OpenWindow {
+            window_id,
+            window_type: String::from("minecraft:crafting_table"),
```

Comment:
The type of `window_type` should probably be changed to `&'static str`, since we don't require dynamic strings here.

Reply:
That causes issues with `Packet`'s `read_from` method, since I cant `try_get_str` and even if I could, it wouldn't be static